### ISSUE TITLE
[QUESTION] Is there any benefit on datasets (small object detection task) with known labels?

### ISSUE THREAD

**Issue by dantetemplar** [lightly-ai/lightly-train#27]
Do I get benefits when:
1. Pretrain with YOLO12 on images with lightly_train
2. Finetune trained model on the very same data but with common pipeline

I have dataset similar to this:
![Image](https://github.com/user-attachments/assets/981d091f-8c44-4b5e-9527-4c20d30f0944)
https://datasetninja.com/lacmus-drone-dataset

60k-80k number of images, up to 4k resolution.

**Comment by dantetemplar** [comment]
Answered here: https://github.com/lightly-ai/lightly/issues/1817#issuecomment-2813122076
